convert2rhel/checks.py

```diff
@@ -67,6 +73,84 @@ def perform_pre_ponr_checks():
     ensure_compatibility_of_kmods()
 
 
+def check_convert2rhel_latest():
+    """Make sure that we are running the latest version of convert2rhel"""
```

Comment:
Also, add this log message here:
`logger.task("Prepare: Checking if this is the latest version of convert2rhel")`

Reply:
And, skip the check if the internet connection is not available to prevent log messages like:
```
19:19:29         out: [07/20/2022 19:19:29] DEBUG - Calling command 'repoquery --releasever=7 --setopt=reposdir=/var/lib/convert2rhel/convert2rhel_repo.DiUuQs convert2rhel'
19:19:30         out: WARNING - Couldn't check if this is the latest version of convert2rhel
19:19:30         out: repoquery failed 1, failure: repodata/repomd.xml from convert2rhel: [Errno 256] No more mirrors to try.
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
19:19:30         out: https://cdn.redhat.com/content/public/convert2rhel/7/x86_64/os/repodata/repomd.xml: [Errno 14] curl#7 - "Failed connect to cdn.redhat.com:443; Connection refused"
```
Thanks, @r0x0d, for spotting that.